docs/guides/cli_overview.md

```
@@ -89,23 +89,23 @@ dynamo build hello_world:Frontend
 ### `deploy`
 
-The `deploy` commmand creates a pipeline on Dynamo Cloud using parameters at the prompt or using a YAML configuration file. For details, see [Deploying Inference Graphs to Kubernetes](dynamo_deploy/README.md).
+The `deploy` commmand creates a dynamo graph on Dynamo Cloud using parameters at the prompt or using a YAML configuration file. For details, see [Deploying Inference Graphs to Kubernetes](dynamo_deploy/README.md).
 
 **Usage**
 ```bash
-dynamo deploy [PIPELINE]
+dynamo deploy [DYNAMO_GRPAH_PACKAGE]
 ```
 
 **Arguments**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)